atom/browser/net/adapter_request_job.h

```diff
@@ -10,6 +10,7 @@
 #include "base/memory/weak_ptr.h"
 #include "net/url_request/url_request_job.h"
 #include "net/url_request/url_request_job_factory.h"
+#include "atom/common/node_includes.h"
```

Comment:
We should include `v8.h` here, `node_includes.h` should only be used when you use APIs in `node.h`, because `node_includes.h` did many hacks to make it possible.